axelrod/graph.py

```diff
@@ -120,25 +122,42 @@ def cycle(length, directed=False):
     return Graph(edges=edges, directed=directed)
 
 
-def complete_graph(size, loops=True):
-    """ Produces a complete graph of specificies size.
-
-    See https://en.wikipedia.org/wiki/Complete_graph for details.
+def complete_graph(size, loops=True, directed=False):
+    """
+    Produces a complete graph of size `length`.
+    https://en.wikipedia.org/wiki/Complete_graph
 
     Parameters
     ----------
     size: int
         Number of vertices in the cycle
     loops: bool, True
-        Should the graph contain cycles?
+        attach loops at each node?
+    directed: bool, False
+        Is the graph directed?
 
     Returns
     -------
     a Graph object for the complete graph
     """
     edges = [(i, j) for i in range(size) for j in range(i + 1, size)]
-    graph = Graph(edges=edges, directed=False)
+    graph = Graph(directed=directed, edges=edges)
+    if loops:
+        graph.add_loops()
+    return graph
+
 
+def attached_complete_graphs(length, loops=True, directed=False):
+    edges = []
+    # Two complete graphs
```

Comment:
I'm not entirely sure I follow what this method is for.

Reply:
It's for my speciation project, just another topology. Can move it out if you want but it would be good to have a few more. I have a grid function that I want to add as well.

Reply:
Could you add a docstring just highlighting the purpose.